src/app/GitUI/UserControls/FileStatusSorter.cs

```diff
@@ -0,0 +1,140 @@
+﻿using GitExtensions.Extensibility.Git;
+using GitExtUtils.GitUI;
+
+namespace GitUI.UserControls;
+
+internal sealed class FileStatusSorter : IFileStatusSorter
```

Comment:
> The name of the class confuses me, it when sorting is by status Maybe just `FileStatusSorting`?

1. It is unlikely that there will be another implementation of `IFileStatusSorter`.
  Since the class' purpose is the creation of a (sub-)tree ordered by file paths, a more specific name could be chosen.
  ("Sorting" would be the wrong type of noun IMO.)
  `IGitItemStatusSorter.CreateTreeSortedByPath` f.i.?
2. Perhaps, the context menu item should rather be renamed to "Group by".